app/src/main/res/layout/screen_teleconsult_share_prescription.xml

```diff
@@ -176,45 +189,38 @@
 
           <TextView
             android:id="@+id/medicalRegistrationIdTextView"
-            style="@style/Clinic.V2.TextAppearance.Body2Left.Grey1"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
+            android:textAppearance="?attr/textAppearanceBody2"
+            android:textColor="?attr/colorHighEmphasis"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintTop_toBottomOf="@id/signatureImageView"
             tools:text="Medical Registration ID: 637835171" />
 
         </androidx.constraintlayout.widget.ConstraintLayout>
 
-      </androidx.cardview.widget.CardView>
+      </com.google.android.material.card.MaterialCardView>
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 
   </androidx.core.widget.NestedScrollView>
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:id="@+id/buttonFrame"
-    style="@style/Clinic.V2.BottomAlignedButtonFrame"
+    style="@style/Widget.Simple.Button.Frame"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@color/blue2"
-    android:clipChildren="false"
-    android:clipToPadding="false"
```

Comment:
Were these removed for a reason?

Reply:
Yes, those are applied in `Widget.Simple.Button.Frame` style now.